torchrec/distributed/quant_embeddingbag.py

```diff
@@ -224,6 +224,7 @@ def __init__(
         self._is_weighted: bool = module.is_weighted()
         self._lookups: List[nn.Module] = []
         self._create_lookups(fused_params, device)
+        self._fused_params = fused_params
```

Comment:
```suggestion
        self.fused_params = fused_params
```

looks like this attr is used externally, remove the _